src/server/api/routes/place.js

```diff
@@ -18,7 +18,9 @@ router.get('/:placeId', function (req, res) {
     const queryingPlace = placeQueryModel.getPlaceData(req.params.placeId);
     queryingPlace
         .then(function (results) {
-            res.json(results[0]);
+            const placeData = Object.assign({}, results[0]);
+            placeData.images = [placeData.image_link];
+            res.json(placeData);
         })
         .catch(function (err) {
             res.status(500).end(`Unable to get place data due to ${err.message}`);
```

Comment:
We should not show the user this detail on the error message. Sometimes this error message originates from database call, and it will reveal things like our table names, the code that threw the error, our db location, etc. 

This is information an API client should not have

Reply:
Okay, but this should also be another PR.

Reply:
Error handling can be a PR of it's own, but can just remove the err.message for now?